src/assets/scss/components/theme-switcher.scss

```
@@ -22,13 +22,17 @@
 
 .theme-switcher__button {
 	box-shadow: var(--shadow-xs);
-	padding: 0.625rem 0.875rem;
+	padding: 0.625rem 0.675rem;
 	display: inline-flex;
 	align-items: center;
 	margin: 0;
 	gap: 0.25rem;
 	color: inherit;
 
+	@media (max-width: 1100px) {
+		padding: 0.625rem 0.375rem;
+	}
```

```suggestion
    @media screen and (min-width: 1121px) and (max-width: 1200px) {
		padding: 0.625rem 0.675rem;
	}

	@media screen and (min-width: 1023px) and (max-width: 1120px) {
		padding: 0.625rem 0.375rem;
	}
```

The theme selection option was still extending beyond the screen and getting cut off, so I made some adjustments.